certified-connectors/Encodian/apiDefinition.swagger.json

```diff
@@ -15,6 +15,20 @@
       "url": "https://www.encodian.com/products/encodian-for-microsoft-power-automate/"
     }
   },
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://www.encodian.com"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://support.encodian.com/hc/en-gb/articles/360010885513-Privacy-Policy"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "Content and Files; Collaboration"
```

Comment:
@Encodian Can you please remove the space between two values - like below
`"propertyValue": "Content and Files; Collaboration"`

Reply:
@Encodian Can you please resolve the comment, so we can approve the PR